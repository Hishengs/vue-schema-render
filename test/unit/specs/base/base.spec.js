import { mount } from '@vue/test-utils';
import { SchemaRender as Render } from '@/index.ts';

const schema = {
  components: [
    {
      type: 'text',
      label: 'test',
      key: 'test',
      value: ''
    }
  ]
};

describe('Base', () => {
  test('default view', () => {
    const wrapper = mount(Render, {
      propsData: {
        schema,
        data: {
          test: 'hello'
        }
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
