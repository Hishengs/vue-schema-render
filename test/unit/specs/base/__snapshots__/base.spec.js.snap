// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base default view 1`] = `
<div class="vsr_component_schema-render">
  <div class="vsr_component_form">
    <form class="el-form el-form--label-top">
      <div class="el-form-item">
        <label for="test"
               class="el-form-item__label"
        >
          <span>
            test
          </span>
        </label>
        <div class="el-form-item__content">
          <transition-stub name="el-zoom-in-top">
          </transition-stub>
        </div>
      </div>
    </form>
  </div>
</div>
`;
